dicom_numpy/combine_slices.py

```diff
@@ -158,7 +158,7 @@ def _validate_slices_form_uniform_grid(slice_datasets):
         'SamplesPerPixel',
         'PixelSpacing',
         'PixelRepresentation',
-        'BitsAllocated',
+        'BitsAllocated'
```

Comment:
I think we should keep this comma, according to PEP8's recommendation:

https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas

Reply:
Right, interesting